fleet_vehicle_additional_fuel_type/i18n/pt_BR.po

```diff
@@ -0,0 +1,68 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* fleet_vehicle_additional_fuel_type
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-04-01 03:42+0000\n"
+"PO-Revision-Date: 2021-04-01 00:43-0300\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"X-Generator: Poedit 2.4.1\n"
+"Last-Translator: Marcel Savegnago <marcel.savegnago@gmail.com>\n"
+"Language: pt_BR\n"
+
+#. module: fleet_vehicle_additional_fuel_type
+#: selection:fleet.vehicle,fuel_type:0
+msgid "Diesel"
+msgstr "Diesel"
+
+#. module: fleet_vehicle_additional_fuel_type
+#: selection:fleet.vehicle,fuel_type:0
+msgid "Electric"
+msgstr "Elétrico"
+
+#. module: fleet_vehicle_additional_fuel_type
+#: selection:fleet.vehicle,fuel_type:0
+msgid "Ethanol"
+msgstr "Ethanol"
```

Comment:
Translate Ethanol to Etanol.
msgstr "Etanol"

Reply:
I removed the translation files because the .pot itself is in gitignore and I ended up forcing it to be included in git. For now, I will wait for the return of @ivan if it is automatically generated and if it is, I will leave it to translate through the weblate.

Reply:
Sorry I just read this!
Yes, they are generated by Travis 😊